src/main/webapp/aboutus.html

```diff
@@ -41,17 +41,11 @@ <h2>Pardis Khodadoustan</h2>
       <li>Aspirational Hobby: Making photo books :D </li>
       <li>Ask me About: Editing photos and painting </li>
     </ul>
-    <h2>Teammate C Name</h2>
```

Comment:
Priyal has also changed Teammate C, so I would suggest changing Teammate A.

Reply:
Sure, I will update it! Thanks!